NEWS.md

```diff
@@ -131,7 +131,7 @@ see the [tag notes](https://github.com/jump-dev/MathOptInterface.jl/releases/tag
 - Improved performance of `Utilities.ScalarFunctionIterator`
 - Added support for `compute_conflict` to MOI layers
 - Added test with zero off-diagonal quadratic term in objective
-- Fixed double deletion of nested bridged `SingleVariable`/`VectorOfVariables`
```

Comment:
all the changes starting from this line should probably be reverted, since they correspond to previous NEWS.md